.github/workflows/make.yml

```diff
@@ -0,0 +1,33 @@
+name: make
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
prob should add 3.7 as a test case since the package allows python >=3.7 (and could add 3.9 as well) - just need to copy the `if` condition for the upload coverage step at the end from truman